### DISCUSSION TITLE
Divisão de Trabalho

### DISCUSSION THREAD

**Discussion by LoriaLawrenceZ** [OZimbres/FIAP-Cap5-Grupo2#7] [Ideas]
# Divisão de Trabalho

Deixo aqui minha sugestão, qualquer coisa comentem!

acho que seria legal, nesta primeira atividade, cada um ficar com uma parte do projeto, visto que é apenas uma única página.

Sugestão da divisão de conteúdo:

| Seção | Issue Correspondente | Responsável |
| :---: | :---: | :---: |
| Header | Barra de Navegação ( #1 ) | `Diego` |
| Loja Virtual | Cards Loja Virtual ( #5 ) | `Felipe` |
| Serviços | Cards Serviços ( #9 ) | `Leon` |
| Footer | Footer ( #8 ) | `Lorenzo` | 

## Editável

Seção para quem quiser editar o post, façam aqui!

Aqui pode colocar qm acham que deve ficar com o que, ou com o que querem fazer!

### 🟢Felipe

Eu gostaria de fazer a Loja Virtual se estiverem de acordo (Com o auxilio de alguém com mais experiência)

| `Felipe` | Loja Virtual ( #5 ) | 
| :---: | :---: |

**Comment by LoriaLawrenceZ** [comment]
Bom, agora que temos 5 integrantes, quais as sugestões?

**Minha *sugestão*** para divisão de trabalho:

| Responsável | Responsabilidade |
| :---: | :---: |
| `Diego` | `Codificação focada` (Header por exemplo) |
| `Felipe` | `Codificação focada` (Cards loja virtual por exemplo) |
| `Leon` | `Codificação focada` (Cards serviços por exemplo) |
| `Lorenzo` & `Vyctoria` | `Mentoria` e `codificação simples` (Footer por exemplo) |

### 🟢Lorenzo

Eu me voluntario a fazer a Barra de Navegação ou o Footer

| `Lorenzo` | Barra de navegação ( #1 ) | Footer ( #8 ) |
| :---: | :---: | :---: |

***EDIT:*** Foi definido que ficarei com a `mentoria` e o desenvolvimento de ----- (eu quero programar 😔)

<details close>
    <summary>Menções</summary>
    @LoriaLawrenceZ @leonvfilho @vyctoriak @TheBaltarO @DSid2024 
</details>

**Reply by DSid2024** [reply]
Seguindo a linha do Lorenzo no WhatsApp, como ele faria o footer, posso ficar com o head então

**Reply by TheBaltarO** [reply]
Boa, pode subir oq fez até o mesmo pra gente dar sequência? Inclusive se ficou algum erro ou duvida, pode escrever uma nota e os experientes analisam e nos passam um feedback.
